src/olympia/addons/models.py

```diff
@@ -825,8 +825,13 @@ def compatible_version(self, app_id, app_version=None, platform=None,
                   u'OS:%s, Mode:%s' % (self.id, app_id, app_version, platform,
                                        compat_mode))
         valid_file_statuses = ','.join(map(str, self.valid_file_statuses))
-        data = dict(id=self.id, app_id=app_id, platform=platform,
-                    valid_file_statuses=valid_file_statuses)
+        data = {
+            'id': self.id,
+            'app_id': app_id,
+            'platform': platform,
+            'valid_file_statuses': valid_file_statuses,
+            'channel': amo.RELEASE_CHANNEL_LISTED,
```

Comment:
Thankfully no. I considered moving it to `legacy_api/utils.py` or something in fact. Think that would be better, for our sanity ?